trl/trainer/dpo_config.py

```diff
@@ -103,7 +103,10 @@ class DPOConfig(TrainingArguments):
         learning_rate (`float`, *optional*, defaults to `1e-6`):
             Initial learning rate for [`AdamW`] optimizer. The default value replaces that of
             [`~transformers.TrainingArguments`].
-        loss_type (`str`, *optional*, defaults to `"sigmoid"`):
+        loss_weights (`dict[str, float]` or `None`, *optional*, defaults to `None`):
+            Use to weight a combination of losses. The keys must be in `loss_type`. By default (if not specified in the dict),
+            the weight for a loss in loss_type is 1.0.
+        loss_type (`str` or `list`, *optional*, defaults to `"sigmoid"`):
```

Comment:
can you also document that, when a list is passed, the loss is the sum of these values?

Reply:
add `"sft"` as well